doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+  executed. In some cases, we may not even know the contributions were funded
+  or done by an employee on work time. (Whether that's the case or not should
+  not change how we approach a contribution).  For grants though, we do expect
+  the PI and funded team to align their work with the project's needs and be
+  receptive to feedback from other NumPy maintainers and contributors.*
+
+
+Principles of spending project funds
+------------------------------------
+
+NumPy will likely always be a project with many times more volunteer
+contributors than funded people. Therefore having those funded people operate
+in ways that attract more volunteers and enhance their participation experience
+is critical. That key principle motivates many of the more detailed principles
+given below for what to pay for and whom to pay.
+
+What to pay for
+```````````````
+
+1. Pay for things that are important *and* otherwise won't get done.
+   *Rationale: there is way more to be done than there are funds to do all
+   those things. So count on interested volunteers or external sponsored work
+   to do many of those things.*
+2. Plan for sustainability. Don't rely on money always being there.
+3. Consider potential positive benefits for NumPy maintainers and contributors,
+   maintainers of other projects, end users, and other stakeholders like
+   packagers and educators.
+4. Think broadly. There's more to a project than code: websites, documentation,
+   community building, governance - it's all important.
+5. For proposed funded work, include paid time for others to review your work
+   if such review is expected to be significant effort - do not just increase
+   the load on volunteer maintainers.
+   *Rationale: we want the effect of spending funds to be positive for
+   everyone, not just for the people getting paid. This is also a matter of
+   fairness.*
+
+When considering development work, principle (1) implies that priority should
+be giving to (a) the most boring/painful tasks that no one likes doing, and to
+necessary structural changes to the code base that are too large to be done by
+a volunteer in a reasonable amount of time.
+
+There are also a large amount of tasks, activities, and projects outside of
+development work that are important and could enhance the project or community
+- think for example of user surveys, translations, outreach, dedicated
+mentoring of newcomers, community organization, website improvements, and
+administrative tasks.
+
+Time of people to perform tasks is also not the only thing that funds can be
+used for: expenses for in-person developer meetings or sprints, hosted hardware
+for benchmarking or development work, and CI or other software services could
+all be good candidates to spend funds on.
+
+Whom to pay
+```````````
+
+1. All else being equal, give preference to existing maintainers/contributors.
+2. Consider this an opportunity to make the project more diverse.
```

Comment:
Although I don't read it that way, 1. and 2. could be considered to be going in different directions. Presumably, there's a point at which you save an order of magnitude of time (and money) by using an internal volunteer of course.

Reply:
@tylerjereddy As you know, the number of current active NumPy volunteers/contributors is very low.
In cases where there is no potential candidate/s from the pool of existing volunteers, the Council is strongly encouraged to advertise an existing paid work opportunity with the underrepresented groups in tech.
We have already established good relationships with DataUmbrella, GeoLatinas, PyLadies, and WiMLDS.

Reply:
Rephrasing point 2 to: _"When looking outside of the current team, consider this an opportunity to make the project more diverse."_